tests/sparseml/transformers/sparsification/modification/conftest.py

```diff
@@ -27,7 +27,11 @@ def mistral_zoo_model():
     stub = "zoo:mistral-7b-evolcodealpaca_mistral_pretrain-pruned50_quantized"
     config = SparseAutoConfig.from_pretrained(stub)
     with init_empty_weights():
-        model = SparseAutoModelForCausalLM.from_config(config)
+        # attn_implementation="eager" needed so that the model uses the
+        # appropriate attention type (MistralAttention)
+        model = SparseAutoModelForCausalLM.from_config(
+            config, attn_implementation="eager"
```

Comment:
Is this something that needs to be applied for every model? Wondering if we should default to this attention implementation